example/sl-prediction/normalize_embedding.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python
+
+# Load required modules
+import sys, os, logging, numpy as np
+this_dir = os.path.dirname(__file__)
+sys.path.append(os.path.join(this_dir, '../../src'))
+
+# CORAL algorithm (Sun, Feng, & Saenko, arXiv:1612.01939v1)
+def coral(D_S, D_T, source_to_target=True, lamb=0.00001):
+    """
+    Arguments:
+    - D_S: Source data (numpy.array)
+    - D_T: Target data (numpy.array)
+    - source_to_target: Transform source into target space (bool)
+    Returns:
+    - D_S^*: Adjusted source data
+    """
+    # Set up
+    from sklearn.preprocessing import scale
+    from scipy.linalg import sqrtm
+    from numpy.linalg import pinv, inv
+    C_S = np.cov(D_S.T) + lamb*np.eye(D_S.shape[1])
+    C_T = np.cov(D_T.T) + lamb*np.eye(D_T.shape[1])
+    C_S_centered = scale(D_S, with_std = False)
+    C_T_centered = scale(D_T, with_std = False)
+    
+    # Perform CORAL (Algorithm 1)
+    if source_to_target:
+        # this is the recommended approach in the coral paper
+        C_S_sqrt = np.real_if_close(sqrtm(C_S))
```

Comment:
Hi Max, where is pipeline taking more time? FYI `sqrtm` is not parallelized so it runs slower than `rkhs_factor` implemented in `handl.py`.

```
>>> timeit.timeit('rkhs_factor(A)', setup=setup_str, number=3)
1.3135006160009652
>>> timeit.timeit('sqrtm(A)', setup=setup_str, number=3)
6.155250631971285
>>> setup_str
'import numpy as np; from scipy.linalg import sqrtm; from handl import rkhs_factor;A = np.random.rand(1000,1000);A = A @ A.T;
```